src/Ball2.jl

```diff
@@ -73,30 +73,38 @@ The ambient dimension of the ball.
 dim(B::Ball2) = length(B.center)
 
 """
-    σ(d::AbstractVector{N}, B::Ball2)::AbstractVector{N} where{N<:AbstractFloat}
+    σ(d::AbstractVector{N}, B::Ball2)::AbstractVector{<:AbstractFloat} where {N<:AbstractFloat}
 
-Return the support vector of a ball in the 2-norm in a given direction.
+Return the support vector of a 2-norm ball in a given direction.
 
 ### Input
 
-- `d` -- a direction
-- `B` -- a ball in the 2-norm
+- `d` -- direction
+- `B` -- ball in the 2-norm
 
 ### Output
 
-The support vector in the given direction. If the given direction has
-norm zero, the origin is returned.
+The support vector in the given direction. If the direction has norm zero, the
+origin is returned.
 
 ### Notes
 
-This function requires computing the 2-norm of the input direction, and this is
-performed in the given precision of the direction's datatype. Exact inputs are not handled.
+Let `c` and `r` be the center and radius of a ball `B` in the 2-norm,
```

Comment:
double backticks?

Reply:
Good question, this is often not clear (and probably inconsistent). One is more code-like and two is more math-like.